automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -62,6 +64,46 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
         }
       }
 
+      "to create, clone, and delete v2 workspaces (using the Resource Buffer), in a v2 billing project" in {
```

Comment:
Opt: This feels like a very chunky test. Would it make sense to break out `create` `delete` and `clone` to their own tests?

Reply:
the main purpose of this test is to verify that deletion works. I'll rename the test to make that clear and I'll also remove the cloning to make it a bit smaller!